src/platforms/common/performance/custom-instrumentation.mdx

```diff
@@ -11,9 +11,16 @@ supported:
   - android
   - ruby
   - java.spring-boot
+  - apple
 description: "Learn how to capture performance data on any action in your app."
 ---
 
+<PlatformSection supported={["apple"]}>
+<Note>
+Attention: This is a preview API available since 7.0.0-alpha.2. This API is not in stable state yet. It may change in the future.
+</Note>
+</PlatformSection>
```

Comment:
```suggestion
<PlatformSection supported={["apple"]}>

<Note>

Attention: This is a preview API available since 7.0.0-alpha.2. This API is not in stable state yet. It may change in the future.

</Note>

</PlatformSection>
```